src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,21 @@
+package com.google.sps.data;
+
+public final class Constants {
+    public static final String REP_ENTITY_TYPE = "Representative";
+    public static final String REP_NAME = "Name";
+    public static final String REP_TITLE = "Official Title";
+    public static final String REP_POSTS = "Posts";
+
+    public static final String POST_ENTITY_TYPE = "Post";
+    public static final String POST_QUESTION = "Question";
+    public static final String POST_ANSWER = "Answer";
+    public static final String POST_REPLIES = "Replies";
+
+    public static final String COMMENT_ENTITY_TYPE = "Comment";
+    public static final String COMMENT_NAME = "Nick Name";
+    public static final String COMMENT_MSG = "Message";
+
+    private static final Logger logger = LogManager.getLogger("Errors");
```

Comment:
as mentioned in the other PR, please make the logger a private static variable in each class where the TAG is the class name.